libraries/cyclestreets-fragments/src/main/java/net/cyclestreets/LocationsFragment.java

```diff
@@ -3,8 +3,8 @@
 import android.content.Context;
 import android.content.Intent;
 import android.os.Bundle;
-import android.support.design.widget.FloatingActionButton;
-import android.support.v4.app.ListFragment;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import androidx.fragment.app.ListFragment;
```

Comment:
Might be worth auto-reordering imports for clarity in this and all the classes you've changed.